Gemfile.lock

```diff
@@ -10,11 +10,12 @@ PATH
 GEM
   remote: http://rubygems.org/
   specs:
-    activesupport (7.0.4)
+    activesupport (6.1.7.3)
```

Comment:
I know this won't make a difference as this lock gets disregarded when the gem is used, if I understand correctly.

Reply:
Yea this is totally fine theoretically, but curious why we're backtracking on the main gemfile?

Reply:
If you look at the most recent main build it is failing on ruby 2.6 because of activesupport 7. I think the better option here would be to exclude `Gemfile` from testing on 2.6.

I will make the change to the tests rather than changing the version. I'm curious how we got ourselves in this state 🤷 .